src/App/App.js

```diff
@@ -52,7 +78,7 @@ class App extends Component {
               <Link to='/favorites'>
                 <h2>View favorites</h2>
               </Link>
-              <Artist artists={this.state.artists} />
+              <Artist artists={this.state.artists} getCardsByArtist={this.getCardsByArtist}/>
```

Comment:
[recommendation] this might benefit from a more specific name than `Artist` since that doesn't semantically explain the responsibility of this component.